app/helpers/application_helper.rb

```diff
@@ -122,4 +122,11 @@ def format_money_without_symbol(number_or_money, options = {})
     options.reverse_merge!(money.default_format_options)
     ActiveSupport::NumberHelper.number_to_delimited(money.amount.round(options[:precision] || 0), { delimiter: options[:delimiter], separator: options[:separator] })
   end
+
+  def format_amount_by_curreny(collection:, money_method:, seperator: " | ", negate: false, options: {})
```

Comment:
```suggestion
  def totals_by_currency(collection:, money_method:, separator: " | ", negate: false)
```

Thoughts on this?  This is doing more than just "formatting", so I think this might describe the output a little more clearly?

Reply:
Agree!